provider/infoblox.go

```diff
@@ -0,0 +1,483 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
```

Comment:
Please use everywhere github.com/sirupsen/logrus, instead of github.com/Sirupsen/logrus

Reply:
👍 As Sandor already wrote this is also currently blocking Travis CI

```
exit status 1: provider/infoblox.go:24:2: cannot find package "github.com/Sirupsen/logrus" in any of:
```

Reply:
Sadly, after complying with this suggestion, I still had issues with glide even after deleting the cache unless I used the trick mentioned [here](https://github.com/sirupsen/logrus/issues/553#issuecomment-306591437). Following that there were still import conflicts. A couple hours later I resorted to sed recursively on the vendor directory.

Reply:
@njuettner where did you see that error? In Travis CI the error I saw was:

```
FAILED   /home/travis/gopath/src/github.com/kubernetes-incubator/external-dns/vendor/github.com/kubernetes/repo-infra/verify/go-tools/verify-gometalinter.sh	1s
```

which to my astonishment happened on master as well.